src/CommunityToolkit/Xamarin.CommunityToolkit/Views/DockLayout.shared.cs

```diff
@@ -0,0 +1,164 @@
+﻿using System;
+using Xamarin.Forms;
+
+namespace Xamarin.CommunityToolkit.UI.Views
+{
+    public enum Dock
```

Comment:
I have moved it to `Dock.shared.cs`. However, I have kept the name.
Why?. Because its use in this way is equal to https://docs.microsoft.com/en-us/dotnet/api/system.windows.controls.dock?view=net-5.0#System_Windows_Controls_Dock_Left
Who already knows WPF, etc. knows this Layout immediately.

Have sense?